packages/tradershub/src/components/Modal/Modal.tsx

```diff
@@ -14,7 +14,8 @@ import ModalHeader from './ModalHeader';
 type TModalChildren =
     | ReactElement<typeof ModalContent>
     | ReactElement<typeof ModalFooter>
-    | ReactElement<typeof ModalHeader>;
+    | ReactElement<typeof ModalHeader>
+    | undefined;
```

Comment:
can't we just use ReactNode to be more flexible 
Supposed to be low level flexible API in case of a modal

Reply:
I think by adding these types we are limiting the children to be only these components, React Node would defeat that purpose. @aizad-deriv